### PR TITLE
Update SAML Request signing certificate download instructions

### PR DESCRIPTION
**Pull request by mmaddex**
It would be great to get someone from engineering to double check these changes, most importantly https://github.com/auth0/docs/commit/887a0b8ec70cc9c7e92b65deaf450a67aa163b9e.

### CODE REVIEW

**Comment by mmaddex** [comment]
I was also curious about the paragraph that reads:

>... when you initiate an authentication request in your application, the IdP will receive that custom domain in your signed request...

Could someone confirm if the "the IdP will receive that custom domain in your signed request" part is accurate? I'm not 100% sure what goes into signing requests. Does the custom domain factor into this, or is it just a combination of the `SAMLRequest` and the key?

Thanks in advanced for the education.

File: articles/custom-domains/additional-configuration.md [hunk 1]
```diff
@@ -123,18 +123,18 @@ If you use Auth0 with a custom domain to issue Access Tokens for your APIs, then
...
+If you want to use SAML identity providers (IdPs) with your custom domain, you must get the service provider metadata from Auth0 (such as `https://<YOUR-CUSTOM-DOMAIN>/samlp/metadata?connection=<YOUR-CONNECTION-NAME>`). This includes updated **Assertion Consumer Service (ACS) URLs**. Then, you have to manually update this value in your IdP(s). This change to your IdP(s) must happen at the same time as you begin using your custom domain in your applications. This can pose a problem if there are multiple IdPs to configure.
 
 Alternatively, you can use signed requests to fulfill this requirement:
 
-- Once your custom domain is set up, go to [Dashboard > Tenant Settings > Custom Domains](${manage_url}/#/tenant/custom_domains) and download the certificate from the link under the **Sign Request** toggle
+- Download the signing certificate from `https://<TENANT>.auth0.com/pem`. Note that `https://<YOUR-CUSTOM-DOMAIN>.com/pem` will return the same certificate.
```
Comment (jeffreylees) [inline articles/custom-domains/additional-configuration.md:137]: Edit: Never mind, I've figured out what I was thinking of

Comment (jeffreylees) [reply articles/custom-domains/additional-configuration.md:137]: Verified that this works ^

**Comment by jeffreylees** [comment]
>Could someone confirm if the "the IdP will receive that custom domain in your signed request" part is accurate? I'm not 100% sure what goes into signing requests. Does the custom domain factor into this, or is it just a combination of the SAMLRequest and the key?

@mmaddex - you might want to direct this question to the product team if you're looking for more specifics? That's where the majority of this copy came from.

**Review by jeffreylees — APPROVED** [review]

File: articles/custom-domains/additional-configuration.md (hunk 1 above)
Comment (mmaddex) [reply articles/custom-domains/additional-configuration.md:137]: sorry... just getting to reading these... glad you figured it out!

**Pull request merged**